modules/340-monitoring-kubernetes/images/kubelet-eviction-thresholds-exporter/loop

```diff
@@ -30,7 +48,7 @@ runtime_root_dir () {
     curl -sS --unix-sock /var/run/docker.sock http://system/info | jq -re '.DockerRootDir'
     ;;
   containerd)
-    crictl --runtime-endpoint unix:/var/run/containerd/containerd.sock info | jq -r '.config.containerdRootDir'
+    dasel -f /var/run/containerd-config.toml '.root' 2>/dev/null || echo "/var/lib/containerd"
```

Comment:
```
root_dir="$(grep "^root" /etc/containerd/config.toml | sed "s/ //g"  | sed "s/\"//g" |cut -d"=" -f2)"
if [-z "${root_dir}" ]; then
  root_dir="/var/lib/containerd"
fi
echo ${root_dir}
```
and we cannot install dasel

Reply:
It could be:
`# root = "/var/lib/containerd"`
`#root = "/var/lib/containerd"`

and many other cases. I guess dasel is cheaper here

Reply:
maybe use something like:
`grep "^root\|^#root\|^# root"`

Reply:
nevermind, this case is closed